LejeuneCohortWeb/Views/Home/About.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@{
+    ViewData["Title"] = "About";
+}
+
+<div class="text-center">
+    <h1 class="display-4">Welcome, MSSA Peers!</h1>
```

Comment:
Change title to be specific to about page!

Reply:
Just updated it with a better description. Thanks for the feedback!